app/models/league.rb

```diff
@@ -21,7 +21,7 @@
 
 class League < Event
   validates :deadline, :startdate, :enddate, presence: true
-  validates :gameday_duration, presence: true
+  validates :gameday_duration, presence: true, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 1000 }
```

Comment:
Should it be possible to have a gameday duration of 0 or are we starting at 1?

Reply:
0 is intended to enable the workaround mentioned in: #342